packages/database-backup-restorer-mariadb/packaging

```diff
@@ -22,6 +22,10 @@ set -e
 MARIADB_VERSION=10.6.3
 MY_DIR=$(pwd)
 
+cp -r /var/vcap/packages/libpcre2/lib/* /usr/local/lib/
+cp -r /var/vcap/packages/libpcre2/bin/* /usr/local/bin/
+cp -r /var/vcap/packages/libpcre2/include/* /usr/local/include
+
```

Comment:
Looks like this approach will work, but it will also pollute the global state. There's a chance we might conflict with other bosh packages that do similar things.

Is there an option to [use LD_LIBRARY_PATH](https://www.hpc.dtu.dk/?page_id=1180) or similar to allow us to use the libpcre directly in `/var/vcap` ?

Reply:
Looks like there might be [an option](https://www.gnu.org/prep/standards/html_node/Configuration.html) to use something like `--libdir` or `--includedir` for `./configure` too...

Reply:
Mariadb is compiled with cmake so not sure I can use the libdir... The LD_LIBRARY_PATH sounds like a better solution! Will take a look